ui/src/pages/server-types/server-types.tsx

```diff
@@ -101,6 +102,7 @@ export const ServerTypes = (): React.ReactElement => {
       updateQuery(payload, {
         onSuccess: async () => {
           queryClient.invalidateQueries({ queryKey: ['app-state'] });
+          window.location.assign(APP_BASE_URL);
```

Comment:
This was previously missed. Without this, the user stays on the server types page after the payload response.